lib/msal-browser/src/index.ts

```diff
@@ -144,6 +144,7 @@ export {
     StringUtils,
     UrlString,
     JsonWebTokenTypes,
+    TimeUtils,
```

Comment:
How critical is it that we export this? Possible to refactor so all usage is contained in msal-browser/common?